androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/journal/triage/previousCare/PreviousCareContent.kt

```diff
@@ -0,0 +1,141 @@
+package com.EENX15_22_17.digital_journal.android.screens.journal.triage.previousCare
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.EENX15_22_17.digital_journal.android.ui.components.*
+import se.predicare.journal.screens.PreviousCareDto
+
+@Composable
+fun previousHealthCare(item: PreviousCareDto) {
```

Comment:
Rendered composables should start with uppercase letter, the IDE should have shown a hint about this